SocialCareCaseViewerApi.Tests/V1/Factories/ResponseFactoryTests.cs

```diff
@@ -342,5 +342,54 @@ public void CanMapPersonDetailsToGetPersonResponse()
 
             result.Should().BeEquivalentTo(expectedResponse);
         }
+
+        [Test]
+        public void MapWarningNoteToResponseReturnsAnObjectWithFormattedDates()
```

Comment:
Could we have an additional test that also makes sure null is handled correctly for any property that is nullable.

We probably want to do this consistently going forward to catch those annoying NullReferenceExceptions.

Reply:
Thanks 🙌🏾 . 
I'll need to merge another PR (#230) before this one to try and fix a bug in Staging. 😅 
I'll try adding a test for checking this after rebasing.